docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.4'
+volumes:
+  npm_data:
+  runtime_data:
+  intl_data:
+
+networks:
+  scratchapi_scratch_network:
+    external: true
+
+services:
+  app:
+    container_name: scratch-www-app
+    hostname: scratch-www-app
+    environment:
+      - API_HOST=http://localhost:8491
+      - FALLBACK=http://scratchr2-app:8080
```

Comment:
Regarding documentation for contributors. Both api and scratchr2 are private, so non-ST folks can't run those containers. Since this docker container assumes that you're running both locally, does this even make sense for a contributor outside the Scratch Team?

Reply:
They don't, but these defaults work for our development environments and I think can be ignored by other folks?